docs/link-manager.md

```diff
@@ -0,0 +1,43 @@
+## Overview
+
+Use the Link Manager instead of an internal router to: 
+
+- route inside micro frontends 
+- reflect the route and 
+- keep the navigation state in Luigi
+
+Here is a code sample and a definition of the different parameters:
+
+````
+// navigation
+LuigiClient.linkManager().navigate('/overview')
+LuigiClient.linkManager().navigate('users/groups/stakeholders')
+LuigiClient.linkManager().fromClosestContext().navigate('/users/groups/stakeholders')
+LuigiClient.linkManager().fromContext('project').navigate('/settings')
+ 
+ 
+// additionally send parameters to route
+LuigiClient.linkManager().fromClosestContext().withParams({foo: 'bar'}).navigate('settings')
+ 
+ 
+// feature: preserve view to go back afterwards
+LuigiClient.linkManager().fromContext('project').navigate('/settings', null, true) // preserve view
+LuigiClient.linkManager().hasBack()
+LuigiClient.linkManager().goBack({foo: 'bar})
+````
+
+#### Navigation
+
+- **navigate(path)** -  Contains either a full absolute path or a relative path without a leading slash that uses the currently active route as a base. This is classical navigation.
```

Comment:
1. Turn the bullet points into full sentences.
2. ~currently~
3. `This is a classical navigation.`